build/yaml/crd/nsx.vmware.com_networkinfoes.yaml

```diff
@@ -0,0 +1,98 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.11.0
+  creationTimestamp: null
+  name: networkinfoes.nsx.vmware.com
+spec:
+  group: nsx.vmware.com
+  names:
+    kind: NetworkInfo
+    listKind: NetworkInfoList
+    plural: networkinfoes
+    singular: networkinfo
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: NetworkInfo is used to report the network info for a namespace.
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          vpcs:
+            items:
+              description: VPCState defines information for VPC.
+              properties:
+                conditions:
```

Comment:
I still think we should put condition in the overall status.
Or could you check whether original K8s resource has specific conditions for each item in the list?